server/platform-support/platform-support.rst

```diff
@@ -741,3 +741,12 @@
 
 .. [#oracle-support] On Oracle Linux, MongoDB only supports the Red Hat
    Compatible Kernel.
+
```

Comment:
This entire paragraph belongs in the Release Notes.

Reply:
Per review comment, will only include in release notes.